packages/components/src/ResourcePicker/Toolbar/Toolbar.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+import NameFilter from './NameFilter';
+import SortOptions from './SortOptions';
+import StateFilter from './StateFilter';
+
+import theme from './Toolbar.scss';
+
+function Toolbar({ name, sort, state }) {
```

Comment:
What about if none of the 3 props are rendered, do we still want to render the wrapper ?